### PR TITLE
Changes to ensure that the website is accessible

### PR DESCRIPTION
**Pull request by souad988**
List all points I checked:

1. Page titles- no issues found
2. Image text alternatives** (alt text)- no issues found
3. Text headings- no issues found
4. Color contrast:Change font color due to low contrast ratio- https://github.com/souad988/Portfolio/compare/portfolio_accessibility?expand=1#diff-4753b2e6427841425517139f09f1342c320338443447af46169dc0f43a0f2cf3L233-R233
5. Resize- no issues found
6. Interaction- no issues found
7. Moving content- no issues found
8. Multimedia- no issues found
9. The basic structure of the page- no issues found

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -230,7 +230,7 @@
```
Comment (medaminedev66) [inline src/styles.css:230]: - The accessibility of your portfolio looks great 😃, but we still have some issues when we zoom the font. Let's make sure that Text, images, and other content do not overlap.

image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @student, :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Please make sure that your pull request is without `linters` errors.
- [ ] Make sure that the title of your portfolio in `<title>` tag briefly describes the content of the page. For example:
- Poor titles:
  - Microverse project | Your portfolio.
  - Portfolio project
- Better page titles:
  - Souad's portfolio
  - Souadfull-stack web developer

_See the inline comments below for more required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles.css (hunk 1 above)
Comment (souad988) [reply src/styles.css:230]: thanks for reviewing , if you can help and give me some hints on how to do that i will be thankful

Comment (medaminedev66) [reply src/styles.css:230]: Hi @souad988,
I would suggest not fixing the height of the elements. Try always using min-height, and assign `auto` for the height. `height: auto`

Comment (souad988) [reply src/styles.css:234]: ok thanks for your help

**Review by brytebee — APPROVED** [review]
Hi, @Team👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



## Optional

- [ ] **Optional:** Your page title is rather too long, I suggest you use a phrase not longer than 3 words. Example SouadElMansouri Portfolio, SouadEl Portfolio, etc.

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**